asistente_ladm_col/gui/supplies/dlg_missing_snc_supplies.py

```diff
@@ -0,0 +1,309 @@
+            if self.custom_feedback.isCanceled():
+                # The algorithm can throw errors even if canceled, so catch them and if we canceled, silent them.
+                return False, 'Missing cobol supplies algorithm canceled!'
+            else:
+                msg = "QgsProcessingException (Missing cobol supplies): {}".format(str(e))
+                self.logger.critical(__name__, msg)
+                self.show_message(msg, Qgis.Critical)
+
+        self.logger.info(__name__, "Missing Cobol Supplies model finished.")
+
+    def package_results(self, output):  
+        for name in output.keys():
+            output[name].setName(name.split(':')[2])
+
+        try:
+            output_geopackage = processing.run("native:package", {'LAYERS': [
+                        output['qgis:refactorfields_1:COMISIONES_TERRENO'],
+                        output['qgis:refactorfields_2:OMISIONES_TERRENO'],
+                        output['qgis:refactorfields_3:COMISIONES_MEJORAS'],
+                        output['qgis:refactorfields_4:OMISIONES_MEJORAS'],
+                        output['qgis:refactorfields_5:COMISIONES_UNID_PH'],
+                        output['qgis:refactorfields_6:OMISIONES_UNID_PH'],
+                        output['qgis:refactorfields_7:COMISIONES_MZ'],
+                        output['qgis:refactorfields_8:OMISIONES_MZ'],
+                        output['qgis:refactorfields_9:COMISIONES_VR'],
+                        output['qgis:refactorfields_10:OMISIONES_VR']],
+                    'OUTPUT': self.gpkg_path,
+                    'OVERWRITE': True,
+                    'SAVE_STYLES': True},
+                                               feedback=self.custom_feedback)
+        except QgsProcessingException as e:
+            if self.custom_feedback.isCanceled():
+                # The algorithm can throw errors even if canceled, so catch them and if we canceled, silent them.
+                return False, 'Package results canceled!'
+            else:
+                msg = "QgsProcessingException (Package layers): {}".format(str(e))
+                self.logger.critical(__name__, msg)
+                self.show_message(msg, Qgis.Critical)
+
+        root = QgsProject.instance().layerTreeRoot()
+        results_group = root.addGroup(QCoreApplication.translate("MissingSncSuppliesDialog", "Results missing supplies"))
+
+        for layer_path in output_geopackage['OUTPUT_LAYERS']:
+            layer = QgsVectorLayer(layer_path, layer_path.split('layername=')[1], 'ogr')
+            self.names_gpkg += '{} '.format(layer_path.split('layername=')[1])
+            if not layer.isValid():
+                return False, QCoreApplication.translate("MissingSncSuppliesDialog", "There were troubles loading {} layer.").format(layer_path.split('layername=')[1])
+            QgsProject.instance().addMapLayer(layer, False)
+            results_group.addLayer(layer)
+            LayerNode = root.findLayer(layer.id())
```

Comment:
`LayerNode` variable name does not follow our code conventions...

Reply:
Completed